test/metabase/task/sync_databases_test.clj

```diff
@@ -19,48 +19,52 @@
          (-> task
              (update :description replace-trailing-id-with-<id>)
              (update :key replace-trailing-id-with-<id>)
-             (update-in [:data "db-id"] class)
              (update :triggers (fn [triggers]
                                  (vec (for [trigger triggers]
-                                        (update trigger :key replace-trailing-id-with-<id>)))))))))
+                                        (-> trigger
+                                            (update    :key  replace-trailing-id-with-<id>)
+                                            (update-in [:data "db-id"] replace-trailing-id-with-<id>))))))))))
 
 (defn- current-tasks []
   (replace-ids-with-<id> (tu/scheduler-current-tasks)))
 
+(defmacro ^:private with-scheduler-setup [& body]
+  `(tu/with-temp-scheduler
+     (do
```

Comment:
`with-temp-scheduler` already puts its `body` inside a `fn` so there's no need to put an additional `do` here